lib/sinatra/authorization.rb

```diff
@@ -40,6 +55,27 @@ def authorize_with_ssl_client
         end
       end
 
+      # Common set of authorization methods used in foreman-proxy
+      def do_authorize
```

Comment:
perform_authorization perhaps?

Reply:
You've seen some old code of mine, so you know I don't mind long names, but I still, prefer a bit shorter variant:

```
["do_authorize_with_ssl_client", "perform_authorization_with_ssl_client"].map(&:size)
=> [28, 37]
```

Let's see if there is anyone else that would choose what version is better?

Reply:
Hmmm I kinda don't like this very generic `do_authorize` because it does not tell anything about how the authorization is performed. I suggest `do_authorize_any` because basically this is the actual semantics.

Reply:
Changed to do_authorize_any